aten/src/ATen/Context.cpp

```diff
@@ -122,6 +122,14 @@ void Context::setAllowTF32CuDNN(bool b) {
   allow_tf32_cudnn = b;
 }
 
+bool Context::allowTF32Onednn() const {
+  return allow_tf32_onednn;
+}
+
+void Context::setAllowTF32Onednn(bool b){
```

Comment:
Pls. rename `b` as readable.

Reply:
We use `b`  here to align to the coding style in this file, like `void Context::setAllowTF32CuBLAS(bool b)` and `void Context::setDeterministicMkldnn(bool b)`